content/docs/command-reference/exp/run.md

```diff
@@ -0,0 +1,97 @@
+# exp run
+
+Run an experiment: reproduce a variation of a committed
+[pipeline](/doc/command-reference/dag) in a hidden project branch.
+
+> Similar to `dvc repro` but for
+> [experimentation](/doc/user-guide/experiment-management).
+
+## Synopsis
+
+```usage
+usage: dvc exp run [-h] [-q | -v] [-f]
+                   [<repro_options> ...]
+                   [--params [<filename>:]<params_list>]
+                   [-n <name>] [--queue] [--run-all] [-j <number>]
+                   [targets [targets ...]]
+
+positional arguments:
+  targets               Stages to reproduce. 'dvc.yaml' by default.
+```
+
+## Description
+
+Provides a way to execute and track experiments in your <abbr>project</abbr>
+without polluting it with unnecessary commits, branches, directories, etc.
+
+> `dvc exp run` is equivalent to `dvc repro` for <abbr>experiments</abbr>. It
+> has the same behavior when it comes to stage execution (restores the
+> dependency graph, etc.). It differs from in that experiment dependencies and
+> outputs will always be <abbr>cached</abbr> (so they can be restored later).
+
+Each `dvc exp run` creates a variation based on the last project version (Git
+commit) and stores it internally with an automatic experiment ID (a short SHA-1
+hash). The results of the last experiment can be seen in the
```

Comment:
> short SHA-1 hash

Is that accurate?

Reply:
Internally experiments are identified by a complete SHA-256 hash (similar to run cache), but I'm not sure we need to document the specifics of it here? The automatic `exp-abc123` naming does come from a shortened version of those hashes, but users can also specify their own names.

Reply:
Ah good to know, thanks!